Corona-Warn-App/src/main/res/values/event_registration_strings.xml

```diff
@@ -284,4 +284,15 @@
     <string name="trace_location_organiser_list_subtitle">"Wählen Sie ein Event, für welches Sie in Vertretung einer infizierten Person andere warnen möchten. Oder stellen Sie gelöschte Events wieder her, indem sie den QR-Code des Events erneut einscannen."</string>
     <!-- XTXT: Event organiser selection list: qr code scann  -->
     <string name="trace_location_organiser_qr_code_scan">"QR-Code scannen"</string>
+
+    <!--  Warn Duration Screen -->
+    <!-- XHED: Warn Duration Screen: header -->
+    <string name="trace_location_organiser_warn_duration_header">"In Vertretung warnen"</string>
+    <!-- XTXT: Warn Duration Screen: body -->
+    <string name="trace_location_organiser_warn_duration_body">"Geben Sie die Aufenhaltsdauer der positiv getesteten Person an, für die gewarnt werden soll. Diese Information erhalten Sie vom Gesundheitsamt."</string>
```

Comment:
On my device, the wrong string is displayed: 
![image](https://user-images.githubusercontent.com/75120552/130040864-c750ec5c-87c1-4c38-bb80-01b6214a5b8a.png)